docs/homeassistant/remote-access/remote-access-through-oracle-cloudflare-tailscale/index.md

```diff
@@ -0,0 +1,475 @@
+<br/>
+
+You can also select which type of instance you need. You can have an x64 type AMD instance, or an Ampere instance. Ampere instances are ARM64 instance, which use processors similar to a raspberry pi; they also have more resources, including more ram. In this case, I am going to use the Ampere (AMR64) instance. 
+<br/>
+
+![Create Compute Instance](../remote-access-through-oracle-cloudflare-tailscale/images/oracle-compute-instance-image.png "Select Instance Image")
+<br/>
+
+Under this configuration, you can choose number of CPUs, and the amount of RAM. For Ampere, you can have a maximum of 4 CPUs, and 24GB of ram. Choose as you require.
+
+
+> 💡 **TIP**: You can spread these resources across multiple instances. For example, you can have 2 instances covering 2 CPUs, and 12GB RAM each.
+
+
+Now, scroll down; at add SSH Keys, you need to add a public key. This is required for managing the instance remotely, via SSH. Click **Paste public keys**, open Putty on your Desktop (Windows) and Generate keys. 
+<br/>
+
+![Paste Public Keys](../remote-access-through-oracle-cloudflare-tailscale/images/oracle-instance-ssh.png "Create instance SSH")
+<br/>
+
+Copy the generated public key from the window and paste in the **SSH Keys** field
+<br/>
+
+![Add SSH Keys](../remote-access-through-oracle-cloudflare-tailscale/images/oracle-instance-ssh1.png "Add SSH Keys")
+<br/>
+
+Now, we need to save the private key, so that we can log-in using this public-private key combination through ssh. If asked for proceeding without a password, press Yes.
+<br/>
+
+![Save SSH Private Keys](../remote-access-through-oracle-cloudflare-tailscale/images/putty-ssh.png "Save SSH Private Keys Putty")
+<br/>
+
+Save this key securely. We will need this key whenever we need to ssh into this instance.
+
+Now, click on **Create** and wait for Oracle to process and provision your instance.
+
+> 💡 **Important things to note**: Once the instance is started and running, note down the public IP of the instance. We will need this to configure our domain access. 
+<br/>
+
+![Instance Overview Page](../remote-access-through-oracle-cloudflare-tailscale/images/oracle-instance-review.png "Oracle Instance Overview")
+<br/>
+
+Now, our Oracle Cloud is up and running. Test your SSH, by loading the key file to PUTTY, and key in the public IP address of the instance. Username is ubuntu and you won't need a password, if using a ssh key file. Check ssh connection, and make sure everything is working properly. 
+
+> :grey_exclamation: **The following configurations assume that you are in an SSH connection with the Oracle Instance**
+
+For the initial configuration, we need to update and upgrade the operating system, and do some housekeeping. We need to run the following commands, one by one.
+
+``sudo apt update; sudo apt upgrade -y``
+
+``sudo apt purge netfilter-persistent iptables-persistent``
+
+``sudo snap remove oracle-cloud-agent``
+
+``sudo apt purge snapd open-iscsi lxd lxcfs``
+
```

Comment:
Can we combine this code block. It would look better as single block rather than multiple broken blocks.